idd/Energy+.idd.in

```diff
@@ -70097,31 +70100,28 @@ HeatPump:WaterToWater:EIR:Heating,
         \minimum> 0.0
         \default 1.0
    A7,  \field Capacity Modifier Function of Temperature Curve Name
-        \note Heating capacity as a function of CW supply temp and entering condenser temp
+        \note Heating capacity modifier as a function of CW supply temp and entering condenser temp
         \required-field
         \type object-list
-        \object-list BiquadraticCurves
-        \object-list BiVariateTables
+        \object-list BivariateFunctions
         \note curve = a + b*CWS + c*CWS**2 + d*ECT + e*ECT**2 + f*CWS*ECT
         \note CWS = supply (leaving) hot water temperature(C)
         \note ECT = entering condenser fluid temperature(C)
    A8,  \field Electric Input to Output Ratio Modifier Function of Temperature Curve Name
-        \note Electric Input Ratio (EIR) as a function of temperature
+        \note Electric Input Ratio (EIR) modifier as a function of temperature
         \note EIR = 1/COP
         \required-field
         \type object-list
-        \object-list BiquadraticCurves
         \object-list BiVariateTables
```

Comment:
BiVariate*Functions*

Reply:
If it's OK, I'll just wait for CI to be happy with this, make this tiny change and let it be merged without re-running CI for it all again.

Reply:
Yeah,sure.